ystia/yorc
# Feature/gh 122 aws volume attachment

# Pull Request description
## Description of the change
Support of EBS(Elastic Block Store) and attachment of EBS to an AWS instance.

### What I did
- Refactoring of "GenerateTerraformInfraForNode()" of generator.go so it can handle the generations of multiples intances correctly. 
- Create "generateEBS()" that can create an EBS with the correct proprerties (zone, size, encryption) and with terraform outputs (volumeid, arn)
- "addAttachedDisks()"  added in aws_instance. It handle the volume attachment by creating a "aws_volume_attachment" ressource.
- Created a nodeParams struct to reduce the number of args in some functions.

### How to verify it
Create a topolgy that have a compute and a block storage.

!! The device name of the block storage must be the same as the name on the instance. !!
/dev/sdf could become /dev/xvdf on most recent Linux system.
[see](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html) 

### Description for the changelog

## Applicable Issues
#Fixes #122
